keps/sig-auth/3257-clustertrustbundles/README.md

```diff
@@ -299,14 +301,16 @@ spec:
 +  - name: example-com-server-tls-trust-anchors
 +    projected:
 +      sources:
-+      - pemTrustAnchors:
-+          clusterTrustBundleName: example.com--server-tls-live
++      - clusterTrustBundle:
++          signerName: example.com/server-tls
```

Comment:
`signerName` needs to be updated to `example.com:server-tls:foo` from the previous example